packages/react-devtools-shared/src/backend/fiber/renderer.js

```diff
@@ -2588,7 +2603,30 @@ export function attach(
   }
 
   function recordSuspenseResize(suspenseNode: SuspenseNode): void {
-    // TODO: Notify the front end of the change.
+    if (__DEBUG__) {
+      console.log('recordSuspenseResize()', suspenseNode);
+    }
+    const fiberInstance = suspenseNode.instance;
+    if (fiberInstance.kind !== FIBER_INSTANCE) {
+      // TODO: Resizes of filtered Suspense nodes are currently dropped.
```

Comment:
Need to figure out how to get IDs for filtered nodes. IDs of filtered instances are hidden on purpose so I need to double check if it's safe to use the ID here.